docs/tutorials/RemovingFields.md

```diff
@@ -7,9 +7,9 @@ title: "Tutorial: Removing Fields"
 >
 > —  Antoine de Saint-Exupery
 
-When working on AddressBook, you will most likely find that some features and fields that are no longer necessary. In scenarios like this, you can consider refactoring the existing `Person` model to suit your use case.
```

Comment:
I think we should delete the tutorial docs from our repo since they are unlikely to have any purpose in the future.

Reply:
I think for this one, may be we can discuss in the next meeting.

Reply:
i agree with yi hsuen. But for final consensus, lets discuss in the meeting as there is no rush to do so